ocs_ci/ocs/ui/base_ui.py

```diff
@@ -426,12 +426,15 @@ def navigate_overview_page(self):
 
         """
         logger.info("Navigate to Overview Page")
-        if Version.coerce(self.ocp_version) >= Version.coerce("4.8"):
+        if Version.coerce(self.ocp_version) > Version.coerce("4.8"):
```

Comment:
Overview page is now ODF starting ODF 4.9, so either the condition is incorrect or the log message

Reply:
If the Overview page is present in ODF 4.9 will update the condition